effect/shared/src/main/scala/scalaz/effect/Fiber.scala

```diff
@@ -37,4 +39,29 @@ trait Fiber[E, A] {
    * successfully interrupted or has produced its result.
    */
   def interrupt[E2](t: Throwable): IO[E2, Unit]
+
+  /**
+   * Monoidal composition of fibers.
+   */
+  final def <>(that: => Fiber[E, A])(implicit M: Monoid[A]): Fiber[E, A] =
+    zipWith(that)(M.append(_, _))
```

Comment:
since this is the monoid instance's `<>`, would it make sense to have one delegate to the other? I think the consensus was for the typeclass instance method to delegate to the class's method, but I'm not sure of that.

Reply:
👍 Class method is faster so makes sense to delegate to that one.